modules/internal/ChapelStandard.chpl

```diff
@@ -24,25 +24,20 @@ module ChapelStandard {
   use startInitCommDiags;  // Internal, but uses standard/CommDiagnostics
 
   // Internal modules.
-  use CPtr;
   use CString;
   use Bytes;
   use String;
   use OwnedObject;
   use SharedObject;
-  use ChapelDebugPrint;
```

Comment:
A bunch of modules also use chpl_debug_writeln, I'm surprised this didn't need more uses too.

Reply:
I thought this was the same thing as with HaltWrappers, but the chpl_debug_writeln() uses aren't fully qualified, so...  that is odd...  Oh, maybe it's because they only use the routine when a config param is true, and we don't typically set those config params to true while running testing?  Adding this to all modules that have chpl_debug_writeln() (or `chpl_debug_*()` wouldn't be too much work.  It was all the other routines that I wasn't familiar with (`chpl__testParWriteln()`, `chpl__testParStop()`, ...) that I didn't want to go looking for if the compiler wasn't complaining about them.